### DISCUSSION TITLE
我在v1.10.17-lts碰到的docker compose删除的问题升级到v1.10.18-lts并未解决!

### DISCUSSION THREAD

**Discussion by XF2886** [1Panel-dev/1Panel#6626] [Q&A]
### 1Panel 版本

v1.10.18-lts

### 问题描述

在v1.10.17-lts碰到了docker compose删不掉模板的问题 , 于是我直接从 /opt/1panel/docker/compose 删掉了相关目录 , 但是编排中仍然显示 , 看到issue中有反馈该问题的 , 等到了 v1.10.18-lts 更新后 , 还是删不掉编排中的记录 , 我手动加回了 /opt/1panel/docker/compose 下的文件夹和docker-compose.yml文件 , 仍无法删除 , 目前编排中有一条僵尸记录 , 无论是否勾选 删除文件 , 删除都提示 请求参数错误: Key: 'ComposeOperation.Path' Error:Field validation for 'Path' failed on the 'required' tag , 直接创建新的同名编排会提示 记录已存在 , 我该如何解决 ?

**Comment by ssongliu** [comment]
是脏数据的问题，你加我，我给你处理一下
![image](https://github.com/user-attachments/assets/8ed61549-b166-499d-b441-dba98150b545)

**Reply by ShineSky96** [reply]
下载/opt/1panel/db下的1Panel.db文件，用SQLite数据库工具打开文件，打开composes这张表，删除要删除的编排名称那条记录，保存后替换原1Panel.db文件即可。
![image](https://github.com/user-attachments/assets/d1d6bd70-2d09-486e-827f-a2dfe24ebf37)
